app/views/announcements/_announcement.html.erb

```diff
@@ -0,0 +1,10 @@
+<% if current_announcement %>
+  <% if current_announcement.start_date <= Date.current && current_announcement.end_date >= Date.current %>
+    <div class="container content-wrapper">
+      <span><%== current_announcement.body %></span>
```

Comment:
Wrap it inside, bootstrap row and column classes. (you won't be required to define much styles in content-wrapper). Also use predefined class "center-row" (find in layouts.scss)

Reply:
clickable links are optional links they can be or not present adjacently to the text if not present i made paragraph to cover full space, grid is making space taken by button static that will look inconsistent so i avoided using grid.

Reply:
Cool. Just add a bit of padding at the bottom for when button moves to the next line. And decrease font for smaller displays
![image](https://user-images.githubusercontent.com/47032027/96690796-c292e000-13a1-11eb-8f54-e93752976948.png)

Reply:
![2020-10-21_22-36](https://user-images.githubusercontent.com/55848322/96755497-6c995900-13f0-11eb-926d-1417508c810c.png)
decreased font size with padding at the bottom